src/commands/System/support.ts

```diff
@@ -0,0 +1,26 @@
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { CommandStore, KlasaMessage } from 'klasa';
+import { MessageEmbed } from 'discord.js';
+import { getColor } from '../../lib/util/util';
+
+export default class extends SkyraCommand {
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			aliases: ['supportserver', 'server'],
```

Comment:
```suggestion
			aliases: ['support-server', 'server'],
```
Nit: small thing to add both `support-server` and `supportserver` as aliases